docs/source/tutorials.rst

```diff
@@ -1,4 +0,0 @@
-From PyTorch to Lightning
-=========================
-
-Talk about how to convert
```

Comment:
Could you pls open an issue, think that I can do it...

Reply:
I deleted it because the Tutorial section already links to the blog post of @williamFalcon 
https://towardsdatascience.com/from-pytorch-to-pytorch-lightning-a-gentle-introduction-b371b7caaf09
which shows how to convert.
I can still open an issue if you think we should have a separate tutorial directly in the docs.

Reply:
well there was a talk about adding notebooks as tutorials and then these notebooks can be generated as standard docs page

Reply:
that would be nice! ok I will open an issue.